docs/how-to/integrations/craft-a-flutter-app.rst

```diff
@@ -33,8 +33,9 @@ Add an app written for Flutter
     :start-at: parts:
     :end-at: flutter-target: lib/main.dart
 
-Flutter parts are built with the `flutter plugin
-<https://snapcraft.io/docs/flutter-plugin>`_.
+.. Missing link to Flutter plugin
```

Comment:
To make it easier to reference this in the future:

```suggestion
.. Missing link to Flutter plugin (SNAPCRAFT-1123)
```

Reply:
5cac99209